src/OpenLoco/src/Graphics/Gfx.cpp

```diff
@@ -316,7 +318,35 @@ namespace OpenLoco::Gfx
     // 0x0046E07B
     void loadCurrency()
     {
-        call(0x0046E07B);
+        const auto* currencyObject = ObjectManager::get<CurrencyObject>();
+        if (currencyObject == nullptr)
+        {
+            return;
+        }
+
+        const auto loadCurrencySymbol = [](const auto font, const auto imageId, const auto dstImageId) {
+            const auto* image = getG1Element(imageId);
+            if (image == nullptr)
+            {
+                return;
+            }
+
+            auto* currencyElement = getG1Element(dstImageId);
+            if (currencyElement == nullptr)
+            {
+                return;
+            }
+
+            Gfx::setCharacterWidth(font, 131, image->width - 1);
```

Comment:
The width is not correct for this it should be:
```
// taken from L174
constexpr std::array<FontEntry, 4> fonts = {
            FontEntry{ Font::medium_normal, -1 },
            FontEntry{ Font::medium_bold, -1 },
            FontEntry{ Font::small, -1 },
            FontEntry{ Font::large, 1 },
 };

...
Gfx::setCharacterWidth(font, u'£', image->width + fontWidthModifier);
```

Reply:
Yeah I missed that the one time its adding subtracting and not adding, I don't think I'll use that table and rather pass an offset as the argument, its less code at the end just having 4 calls.